examples/gno.land/r/demo/teritori/groups/README.md

```diff
@@ -0,0 +1,24 @@
+### - test package
```

Comment:
Same comment as before -- what do you think about converting this into a script?